yt_dlp/extractor/slideslive.py

```diff
@@ -96,6 +123,49 @@ class SlidesLiveIE(InfoExtractor):
         'only_matching': True,
     }]
 
+    _WEBPAGE_TESTS = [{
+        'url': 'https://iclr.cc/virtual_2020/poster_Hklr204Fvr.html',
+        'info_dict': {
+            'id': '38925850',
+            'ext': 'mp4',
+            'title': 'Towards a Deep Network Architecture for Structured Smoothness',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'timestamp': 1629671508,
+            'upload_date': '20210822',
+            'chapters': 'count:7',
+        },
+        'params': {
+            'skip_download': 'm3u8',
+        },
+    }]
+
+    @classmethod
+    def _extract_from_webpage(cls, url, webpage):
+        # Reference: https://slideslive.com/embed_presentation.js
+        for embed_id in re.findall(r'(?s)new\s+SlidesLiveEmbed\s*\([^\)]+\bpresentationId:\s*["\'](\d+)["\']', webpage):
+            url_parsed = urllib.parse.urlparse(url)
+            origin = f'{url_parsed.scheme}://{url_parsed.netloc}'
+            yield cls.url_result(update_url_query(
+                f'https://slideslive.com/embed/presentation/{embed_id}', {
+                    'embed_parent_url': url,
+                    'embed_container_origin': origin,
+                }))
+
+    def _parse_slides_xml(self, xml, video_id):
+        slides = []
+        if not xml:
+            return slides
+        for slide in xml.findall('./slide'):
+            name = xpath_text(slide, './slideName', 'name')
+            if not name:
+                continue
+            slides.append({
+                'id': xpath_text(slide, './orderId', 'id'),
+                'time': int_or_none(xpath_text(slide, './timeSec', 'time')),
+                'url': f'https://cdn.slideslive.com/data/presentations/{video_id}/slides/big/{name}.jpg',
+            })
+        return slides
```

Comment:
Why put it into a custom dict only to unpack it immediately

Reply:
An artifact from an earlier version of the code where it was downloading/parsing the XML slides info and then falling back to the JSON slides info if necessary. But the JSON URL returns a 403 about half of the time whereas the XML is always available, so I simplified by removing the JSON code altogether. Clearly I didn't simplify enough, though

Reply:
welp, I came across a link that *only* has JSON slides info:  https://slideslive.com/38979920/
I will have to re-work this